ksql-common/src/test/java/io/confluent/ksql/serde/ValueFormatTest.java

```diff
@@ -28,7 +28,10 @@
 
 public class ValueFormatTest {
 
-  private static final FormatInfo FORMAT_INFO = FormatInfo.of(AVRO, Optional.of("something"));
+  private static final FormatInfo FORMAT_INFO =
```

Comment:
super nit: style wise, how about:

```java
private static final FormatInfo FORMAT_INFO =  FormatInfo.of(
    AVRO,
    Optional.of("something"),
    Optional.empty()
);
```

Reply:
I think I did make the change - its now split over multiple lines.